controller-server/src/main/java/com/yahoo/vespa/hosted/controller/deployment/DeploymentTrigger.java

```diff
@@ -338,13 +337,13 @@ else if (testJobs.isEmpty()) {
 
     /** Returns whether the given job is currently running; false if completed since last triggered, asking the build service otherwise. */
     private boolean isRunning(Application application, JobType jobType) {
-        return ! application.deploymentJobs().statusOf(jobType)
+        return    ! application.deploymentJobs().statusOf(jobType)
                                .flatMap(job -> job.lastCompleted().map(run -> run.at().isAfter(job.lastTriggered().get().at())))
                                .orElse(false)
-               && jobStateContains(application, jobType, running, queued);
+               &&   jobStateIsAmong(application, jobType, running, queued);
     }
 
-    private boolean jobStateContains(Application application, JobType jobType, JobState... states) {
+    private boolean jobStateIsAmong(Application application, JobType jobType, JobState... states) {
```

Comment:
I guess a solution is to let the jobState function return just that -- the job state. Then this can be checked for equality or membership in a collection as desired. Actually, that's a good idea.